ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -316,7 +314,13 @@ object CommandService {
       message: String,
   ) extends Error
   final case class GrpcError(status: io.grpc.Status) extends Error
-  final case class InternalError(id: Option[Symbol], message: String) extends Error
+  final case class InternalError(id: Option[Symbol], error: Throwable \/ String) extends Error
+  object InternalError {
+    def apply(id: Option[Symbol], message: String): InternalError =
+      InternalError(id, \/.right(message))
+    def apply(id: Option[Symbol], error: Throwable): InternalError =
```

Comment:
What if we just only kept the message of the error here, so `InternalError(id, error.message)`